packages/core/src/mesh/PrimitiveMesh.ts

```diff
@@ -1360,6 +1360,7 @@ export class PrimitiveMesh {
       radius,
       height,
       radialSegments,
+      thetaStart,
```

Comment:
> from context we know thetaStart is a fixed value Math.PI. Do we really need to add it into the function input instead of using the Math.PI directly?

Just to be consistent with the cylinder initialization angle